### DISCUSSION TITLE
undefined method Google\Client::isUsingJwtWithScope()

### DISCUSSION THREAD

**Discussion by georyitt** [freekmurze/laravel-analytics#429] [General]
Call to undefined method Google\Client::isUsingJwtWithScope() error

**Comment by sanjaydeveloper15** [comment]
Hi 👋 
I am getting login required error

I am using json file and view ID both
As per documentation there is no option given where we need to put oauth access token 

So do you have any idea regarding it ?

**Comment by georyitt** [comment]
Hi Sanjay Kumar.

The first thing you have to add in the .env file

ANALYTICS_VIEW_ID = {{view_id}}

Second, you must include the json data in the path storage/app/analytics/service-account-credentials.json
You can see in the config/analytics path file that path and the credential of the .env file.

Third, you must call the library occupies the following import:

use Spatie\Analytics\AnalyticsFacade as Analytics;


Hope this helps you.

De: Sanjay ***@***.***>
Enviado: miércoles, 8 de diciembre de 2021 5:33
Para: ***@***.***>
CC: ***@***.***>; ***@***.***>
Asunto: Re: [spatie/laravel-analytics] undefined method Google\Client::isUsingJwtWithScope() (Discussion #429)


Hi 👋
I am getting login required error

I am using json file and view ID both
As per documentation there is no option given where we need to put oauth access token

So do you have any idea regarding it ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/spatie/laravel-analytics/discussions/429#discussioncomment-1771075>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AGDAH4TRKIHDNHK36PPOB43UP4J63ANCNFSM5JFSCHOA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
